02-Reproject-Raster-In-R.Rmd

```diff
@@ -246,15 +258,16 @@ plot(DTM_HARV,
      legend=F)
 ```
 
-We have now successfully layered the Digital Terrain Model on top of our hillshade
-to produce a nice looking, textured basemap! 
+We have now successfully layered the Digital Terrain Model on top of our
+hillshade to produce a nice looking, textured base map! 
+
+##Challenge
+Create a map of the San Joaquin study site using the `SJER_DSMhill_WGS84.tif`
```

Comment:
Create a map of the <a href="http://www.neoninc.org/science-design/field-sites/san-joaquin-experimental-range" target="_blank">NEON San Joaquin Experimental Range Field Site</a> using...

(we call them field sites)
Also let's link to the SJER field site page!